assembler/manual.html

```diff
@@ -285,11 +285,11 @@ <h3>2.3 Downloading and compiling SPAdes source code</h3>
         <li><code>plasmidspades.py</code>  (main executable script for <a href="#plasmid">plasmidSPAdes</a>)</li>
         <li><code>rnaspades.py</code>  (main executable script for <a href="rnaspades_manual.html" target="_blank">rnaSPAdes</a>)</li>
         <li><code>truspades.py</code>  (main executable script for <a href="truspades_manual.html" target="_blank">truSPAdes</a>)</li>
-        <li><code>hammer</code>  (read error correcting module for Illumina reads)</li>
+        <li><code>hammer</code>  (read error correcting module for Illumina reads using <code>k=21</code>)</li>
```

Comment:
k=21 is a property of BayesHammer, it is not user-configurable and does not make much sense in exposing (the description is in BayesHammer paper and we're feeling it's enough)

Reply:
For me the k-mer size is too low and I think it should be emphasized in the documentation. It is better to use other error-correcting tool, IMHO.